src/explore-education-statistics-common/src/components/Link.tsx

```diff
@@ -0,0 +1,34 @@
+import React, {
+  AnchorHTMLAttributes,
+  FunctionComponent,
+  ReactNode,
+} from 'react';
+
+type Props = {
+  className?: string;
+  to: string;
+  children: ReactNode;
+} & AnchorHTMLAttributes<HTMLAnchorElement>;
+
+type RendererType = FunctionComponent<Props>;
+
+// some default renderer
+let Renderer: FunctionComponent<Props> = ({ children }: Props) => {
```

Comment:
Would just call this `InternalLink`. Render sounds like it's actually going to go do something else

Reply:
It's an internal component within the Link file... it renders.